runtime/stdlib/account.go

```diff
@@ -784,7 +792,12 @@ func newAccountKeysCountConstructor(
 
```

Comment:
Maybe also rename `newAccountKeysCountConstructor` to e.g. `newAccountKeysCountGetter`: it s not providing a constructor in the sense of the other constructors, e.g. like `NewAuthAccountConstructor`, but just a field getter function.

Reply:
objects are just verbose lambdas 🤔